packages/kotti-checkbox/src/Checkbox.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+	<label class="form-checkbox">
+		<input
+			class="form-checkbox__input"
+			v-model="model"
+			:name="name"
+			type="checkbox"
+		/>
+		<i class="form-icon" />
+		<span class="form-checkbox__label">
+			<slot>{{ label }}</slot>
+		</span>
+	</label>
+</template>
+
+<script>
+export default {
+	name: 'KtCheckbox',
+	props: {
+		label: { types: [String] },
+		name: { types: [String] },
+		value: { types: [Boolean] },
+		initialValue: { types: [Boolean] },
+	},
+	computed: {
+		model: {
+			get() {
+				let value
```

Comment:
If you for whatever reason don’t want to remove support for `initialValue`, at least do this:

```suggestion
				return this.value === null ? this.initialValue : this.value
```

Reply:
The initial value was asked by @amrdraz.
Anyway I think your suggestion differs too much from the original goal. Just imagine value being `"I'm a String"`. Plus the style here is explicit does what it seems to do, even if it needs few lines for it.

Reply:
<img width="411" alt="Screen Shot 2019-04-17 at 11 44 49 AM" src="https://user-images.githubusercontent.com/15861247/56278379-642ed680-6106-11e9-939c-4b02f4d47514.png">

Reply:
Yes, vue is unfortunately not 100% strict about this. However, you’re misusing the API and you’re getting a warning in the console that strings aren’t allowed there. Should we also support `MediaStream` values? 🙂